pkg/cluster/internal/providers/podman/provision.go

```diff
@@ -185,6 +185,9 @@ func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, n
 		// including some ones podman would otherwise do by default.
 		// for now this is what we want. in the future we may revisit this.
 		"--privileged",
+		// For using podman when VirtualBox is also installed
+		// See https://github.com/containers/podman/issues/14284
```

Comment:
this seems to be needed only for rootless, if you see in the `commonArgs`method we append optional arguments based on some environment details, we should have a check for rootless somewhere so we can use the same pattern for this option instead of setting it for everything